src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFileExtensions.ZipArchive.Create.cs

```diff
@@ -36,6 +36,7 @@ public static partial class ZipFileExtensions
         /// <exception cref="FileNotFoundException">The file specified in sourceFileName was not found. </exception>
         /// <exception cref="NotSupportedException">sourceFileName is in an invalid format or the ZipArchive does not support writing.</exception>
         /// <exception cref="ObjectDisposedException">The ZipArchive has already been closed.</exception>
+        /// <param name="destination">The zip archive to add the file to.</param>
```

Comment:
I think this line of code should be placed on the next line. Because according to the previous format, the exception and parameter are separated by a blank line of comments